pkg/controllers/deprovisioning/drift.go

```diff
@@ -71,7 +71,6 @@ func (d *Drift) ComputeCommand(ctx context.Context, nodes ...*Candidate) (Comman
 	}); len(empty) > 0 {
 		return Command{
 			candidates: empty,
-			action:     actionDelete,
```

Comment:
I know I'm late to the review party here, but I wonder if thing would be a little cleaner if `func (d *Drift) ComputeCommand` returned `(*Command, error)` (instead of `(Command, error)`, and we could simply return `nil` when we determine that there's nothing to do. This seems more expressive than maintaining two separate consts for "nothingness" (one const for "empty candidates", one const for "don't do anything").

Thoughts?

Reply:
Not sure I understand what you mean by having two consts for "nothingness". We only have one const representing don't do anything which is if the len(candidates) == 0. 

On the topic of `*Command` vs `Command`,  I think it looks cleaner with `nil, nil` return values, but with `Command{}, nil` it more resembles the `reconcile.Result{}, nil`, with the empty structs both referring to nothing to be done.

Reply:
Right, I mistook the `empty` var there for a static representation of "emtpy" but I now see that in this case it's just a var that wraps an empty slice of type `*Candidate`.

In any event explicitly returning a struct literal with an empty slice, and then adding a method onto that type struct to return data based on no data (the new `.Action()` method that we're using to detect "don't do anything" seems like more work than `== nil` checks.

Food for thought!